packages/time/__tests__/semester.test.js

```diff
@@ -0,0 +1,59 @@
+// @ts-check
+
+import {expect} from 'chai';
+import Semester from '../lib/semester.js';
+
+const verifyReadable = (semesterString, year, season) => {
```

Comment:
I would go with `assertParsedSemester` or `expectParsedSemester` and define it within the test case

Reply:
I proposed both renaming and moving, not sure if you saw the moving part 🙂

Reply:
I did in fact miss it. Moved it now